package.json

```
@@ -204,5 +204,9 @@
             "@types/react": "^18.0.18",
             "webpack": "5.84.1"
         }
+    },
+    "engines": {
+        "node": ">=16",
```

```suggestion
        "node": ">=18",
```
Refer: https://pnpm.io/installation#compatibility